composer.json

```diff
@@ -8,7 +8,6 @@
     "require": {
         "php": ">=8.1",
         "cweagans/composer-patches": "^1.7 || ^2.0",
-        "drupal/allowed_formats": "^2.0 || ^3.0",
```

Comment:
After trying to upgrade to EWPP-3795 branch I have the following error:
`Module allowed_formats has an entry in the system.schema key/value storage, but is missing from your site. [More information about this error](https://www.drupal.org/node/3137656).`
Should we also here still keep that module as required in composer.json (and do removal in other release).

Reply:
Ok, I added it back.